internal/build/build.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 var (
-	Version              = "Version not set"
+	Version              = "version not set"
 	GitRevision          = "not set"
 	GitBranch            = "not set"
 	uiVersionDefaultText = "Viewer not build"
```

Comment:
Since we are fixing the Casing, could we fix it here as well. Thank you. 
"Viewer not built" -> "viewer not built"

Reply:
missed that one, my bad.